src/components/DAOs.tsx

```diff
@@ -1,72 +1,59 @@
 import * as React from "react";
 import { Observable } from "rxjs";
-import {
-  CProps,
-  ComponentList,
-  ComponentListProps
-} from "../runtime";
-import {
-  Arc as Protocol,
-  ArcConfig as ProtocolConfig
-} from "../protocol";
-import {
-  ArcDAO as Component,
-  DAOEntity as Entity,
-  DAOData as Data
-} from "./";
-import {
-  IDAOQueryOptions as FilterOptions
-} from "@daostack/client";
+import { CProps, ComponentList, ComponentListProps } from "../runtime";
+import { Arc as Protocol, ArcConfig as ProtocolConfig } from "../protocol";
+import { ArcDAO as Component, DAOEntity as Entity, DAOData as Data } from "./";
+import { IDAOQueryOptions as FilterOptions } from "@daostack/client";
 
-interface RequiredProps extends ComponentListProps<Entity, Data, FilterOptions> { }
+type RequiredProps = ComponentListProps<Entity, Data, FilterOptions>;
```

Comment:
All these changes was done by `eslint fix`. Do you think we should rollback? @dOrgJelli

Reply:
No, I think this one actually makes sense and is an improvement :P